libavutil/hwcontext_vulkan.c

```diff
@@ -2342,7 +2342,8 @@ static int vulkan_map_from_drm_frame_desc(AVHWFramesContext *hwfc, AVVkFrame **f
                      hwfc->sw_format, src->width, src->height, i);
 
         for (int j = 0; j < planes; j++) {
-            plane_data[j].offset     = desc->layers[i].planes[j].offset;
+            plane_data[j].offset     =
+                desc->layers[i].planes[j].offset - desc->layers[i].planes[0].offset;
```

Comment:
AVDRMPlaneDescriptor has the member of object_index, which means different planes may have different objects, so 'desc->layers[i].planes[j].offset - desc->layers[i].planes[0].offset' doesn't make sense to me.

Reply:
I am not aware of that. Thank you.
According to the function BindImageMemory2() below, vulkan assume one image (one layer) should be in one drm object. I will add a check here.

Reply:
The offset in VkSubresourceLayout and the offset in VkBindImageMemoryInfo is confusing to me. 
According to spec, the plane_data[j].offset is relative to the base address of plane and the drm's offset is used to bind memory below. They are different. I think the offset here should be 0. At least setting to 0 can get correct output. 
I need more investigation.
Thanks for pointing this out .

Reply:
According to spce, if it is single planer, this is offset to image and it should be zero because it has only on plane. If it is multi-planer and we use VkBindImagePlaneMemoryInfo to bind memory, the image should be disjoint and the offset is relative to the plane and should be 0 as well.